README.md

```diff
@@ -112,12 +112,12 @@ t.Log(people)
 Installation
 
 ```bash
-go get -u "github.com/scylladb/gocqlx/v3/cmd/schemagen"
+go install github.com/scylladb/gocqlx/v3/cmd/schemagen@latest
```

Comment:
I wanted to commit the v2->v3 change, but I realized that this would no longer work ref: https://github.com/scylladb/gocqlx/issues/287
@dkropachev what do you think? Should we have your workaround mentioned in the README?

Reply:
Yes, could you please do it.